Lib/heapq.py

```diff
@@ -599,9 +599,3 @@ def nlargest(n, iterable, key=None):
     from _heapq import _heappop_max
 except ImportError:
     pass
-
-
-if __name__ == "__main__":
```

Comment:
Yep, will do.

* Is the rest of the change (to run the doctests as part of the test suite) useful or should I just revert the commit?
* Is there anything I should do to mark these lines as uncovered so that coveragepy reports 100%? (I looked, but the other modules with this snippet like difflib and pickle also don't have 100% coverage yet.)